### PR TITLE
DO NOT MERGE - TO MAKE SURE CI DOES NOT HANG

### PR DESCRIPTION
**Pull request by jacobowitz**
just using this for investigating hanging CI tests, please dont review

### CODE REVIEW

File: jina/peapods/runtimes/zmq/zed.py [hunk 1]
```diff
@@ -251,24 +251,33 @@ def _handle(self) -> 'ZEDRuntime':
...
+            self.logger.info('DONE IDLE')
         elif self.request.command == 'CANCEL':
+            self.logger.info('DO CANCEL')
             if self.envelope.receiver_id in self._idle_dealer_ids:
                 self._idle_dealer_ids.remove(self.envelope.receiver_id)
+            self.logger.info('DONE CANCEL')
         elif self.request.command == 'ACTIVATE':
+            self.logger.info('DO ACTIVATE')
             self._zmqstreamlet._send_idle_to_router()
+            self.logger.info('DONE ACTIVATE')
```
Comment (JoanFM) [inline jina/peapods/runtimes/zmq/zed.py:276]: just as warning this is confusing, these `send_idle_to_router` and `Send_cancel_to_router` I do not think wait for a response from `router`

File: jina/peapods/peas/__init__.py [hunk 2]
```diff
@@ -57,7 +57,9 @@ def run(
     :param is_ready: concurrency event to communicate runtime is ready to receive messages
     :param cancel_event: concurrency event to receive cancelling signal from the Pea. Needed by some runtimes
     """
+    print(f'running pea {name}', flush=True)
```
Comment (hanxiao) [inline jina/peapods/peas/__init__.py:60]: looks verbose

File: jina/peapods/peas/__init__.py [hunk 3]
```diff
@@ -57,7 +57,9 @@ def run(
...
     logger = JinaLogger(name, **vars(args))
+    logger.info('logger created')
```
Comment (hanxiao) [inline jina/peapods/peas/__init__.py:62]: looks verbose

File: jina/peapods/peas/__init__.py [hunk 4]
```diff
@@ -96,6 +98,7 @@ def _set_envs():
             runtime.run_forever()
     finally:
         _unset_envs()
+        logger.info(f'{name} set shutdown')
```
Comment (hanxiao) [inline jina/peapods/peas/__init__.py:101]: looks verbose

**Comment by JoanFM** [comment]
@hanxiao this is not to be merged. This was helping us debug problems with `CI`. And we are keeping it alive to make sure we do not see the errors again

**Comment by maximilianwerk** [comment]
Can we close? @jacobowitz @JoanFM

**Comment by JoanFM** [comment]
Yes, I think we are quite confident now

**Pull request closed without merging**